docs/en/operations/settings/settings.md

```diff
@@ -365,7 +365,20 @@ throws an exception.
 
 ## input_format_null_as_default {#settings-input-format-null-as-default}
 
-Enables or disables using default values if input data contain `NULL`, but the data type of the corresponding column in not `Nullable(T)` (for text input formats).
+Enables or disables insertion of default values if input data contain `NULL`, but the data type of the corresponding column in not `Nullable(T)` (for text input formats). This setting is applied to `INSERT ... VALUES` queries.
+
+## insert_null_as_default {#insert_null_as_default}
+
+Enables the insertion of [default values](../../sql-reference/statements/create/table.md#create-default-values) instead of [NULL](../../sql-reference/syntax.md#null-literal) into non-[Nullable](../../sql-reference/data-types/nullable.md#data_type-nullable) columns. This setting is applied to `INSERT ... SELECT` [queries](../../sql-reference/statements/insert-into.md#insert_query_insert-select). `SELECT` subqueries may be concatenated with `UNION ALL` clause.
```

Comment:
```suggestion
Enables the insertion of [default values](../../sql-reference/statements/create/table.md#create-default-values) instead of [NULL](../../sql-reference/syntax.md#null-literal) into non-[Nullable](../../sql-reference/data-types/nullable.md#data_type-nullable) columns. This setting is applicable to `INSERT ... SELECT` [queries](../../sql-reference/statements/insert-into.md#insert_query_insert-select). `SELECT` subqueries may be concatenated with `UNION ALL` clause.
```